app/src/androidTest/java/com/sdp/movemeet/view/home/LoginActivityTest.java

```diff
@@ -25,12 +25,12 @@
 
 public class LoginActivityTest {
 
-    public static final String mEmail = "email";
-    public static final String mPassword = "password";
+    public static final String falseEmail = "email";
+    public static final String falsePassword = "password";
     public static final String shortPassword = "pass";
 
-    public static final String Email = "movemeet@gmail.com";
-    public static final String Password = "password";
+    public static final String trueEmail = "movemeet@gmail.com";
```

Comment:
Even though we have no abstraction for an authentication service, is there no way to use mocks in these tests to avoid having to actually sign in and out?

Reply:
It could be possible in fact to mock the authentication! As you showed us this evening, it is possible to mock almost any class using mockito. I have to investigate this yes!